lib/DueDatePicker/styles.scss

```diff
@@ -0,0 +1,349 @@
+/* ==========================================================================
+   Config
+   ========================================================================== */
+$duedate-popover-width: 304px !default;
+$duedate-timepicker-width: 80px !default;
+$duedate-timepicker-height: 220px !default;
+/* ==========================================================================
+   Styles
+   ========================================================================== */
+.duedate__popover.duedate__popover {
+  max-width: $duedate-popover-width;
+  width: $duedate-popover-width;
+  margin-top: $layout-spacing-base / -2;
+
+  .arrow {
+    display: none;
+  }
+
+  &.duedate__popover .popover-content {
+    padding: 0;
+  }
+}
+
+.duedate__container {
+  font-size: $typo-size-slight;
+  position: relative;
+}
+
+.duedate__button {
+  color: $color-text-light;
+
+  .button {
+    text-decoration: underline;
+    color: $color-text-light;
+  }
+
+  .icon {
+    margin-right: $layout-spacing-base / 4;
+    top: -2px;
+    position: relative;
+    vertical-align: middle;
+  }
+
+  .icon path {
+    fill: $color-link-base;
+  }
+
+  &:hover, &:active {
+    text-decoration: none;
+
+    .icon path {
+      fill: $color-link-hover;
+    }
+  }
+}
+
+.duedate__button--set-duedate {
+  color: $color-link-base;
+}
+
+.duedate__button--overdue {
+  color: $primary-red;
+
+  .button {
+    color: $primary-red;
+  }
+
+  &:hover {
+    color: $primary-red;
+
+    .icon path {
+      fill: $primary-red;
+    }
+  }
+
+  .icon path {
+    fill: $primary-red;
+  }
+
+  .icon {
+    top: -1px;
+  }
+}
+
+.duedate__container--completed {
+  .duedate__button, .duedate__toggle {
+    text-decoration: line-through;
+    font-style: italic;
+    color: $color-text-light;
+  }
+
+  .icon path, &:hover .icon path {
+    fill: $color-text-light;
+  }
+}
+
+.duedate__header {
+  padding: $layout-spacing-base / 2;
+  border-bottom: 1px solid $neutral-light;
+  font-size: $typo-size-slight;
+}
+
+.duedate__header--date {
+  font-size: $typo-size-base;
+}
+
+.duedate__header--time {
+  color: $color-text-base;
+  text-decoration: underline;
+
+  &:hover {
+    color: $color-text-base;
+  }
+}
+
+.duedate__header--overdue .duedate__header--date {
+  color: $primary-red;
```

Comment:
Nothing may need to change here but anywhere with just `color: $primary-red` is an area that may be better suited to use a helper class. Let me know what you think.

Reply:
Doing at times can help reduce the amount of BEM bloat which can happen when you elements have multiple modifiers. One major goal is to make our codebase usable for anyone and a mixture of helper classes and BEM can work really well when balanced in the right scenarios.